.changeset/red-parrots-try.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/draft-table": minor
```

Comment:
This is a `patch` as it doesn't introduce any new features 😄

Reply:
Thanks @gyfchong, that has been updated.